HEC.FDA.Model/paireddata/IPairedDataProducer.cs

```diff
@@ -2,6 +2,7 @@ namespace HEC.FDA.Model.paireddata
 {
     public interface IPairedDataProducer
     {
-        PairedData SamplePairedData(double probability, bool computeIsDeterministic);
+        PairedData SamplePairedData(long iteration, bool computeIsDeterministic);
```

Comment:
Do we need to keep this compute is deterministic?

Reply:
resolved